core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/wait.yml

```diff
@@ -35,6 +35,6 @@
   shell: kubectl get pods --all-namespaces -o json
   register: output
   until: output.stdout|from_json|json_query('items[*].status.conditions[].status')|unique == ["True"]
-  retries: 600
+  retries: 600 # 20min
```

Comment:
Won't be those comments misleading if someone will forget to change number of minutes after changing retries?